examples/Manifest.toml

```diff
@@ -1,13 +1,13 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.10.6"
+julia_version = "1.10.5"
```

Comment:
We usually don't change back the julia version here unless needed. Also there are a lot of changes in the Manifest - are they needed for this PR?